pkgs/misc/vim-plugins/default.nix

```diff
@@ -192,11 +193,11 @@ let
   };
 
   vim-nix = buildVimPluginFrom2Nix { # created by nix#NixDerivation
-    name = "vim-nix-2018-04-24";
+    name = "vim-nix-2018-07-01";
```

Comment:
I made some changes to the indentation handling, let me know if you notice issues.